CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 1.1.3 (Next)
 
 * Your contribution here.
+* [#78](https://github.com/dblock/iex-ruby-client/pull/78): Added `extended_change_percent` and `extended_change_percent_s` properties to Quote. - [@reddavis](https://github.com/reddavis).
```

Comment:
Let's quote `Quote` or maybe just "Added `Quote#extended_change_percent` and `#extended_change_percent_s`.

The extra period in the sentence is throwing danger-changelog off.

Reply:
> I like it.
> 
> We don't round in `change_percent_s`, is there a reason we would here?
> 
> You could just make `percent_s` regular methods and take an rounding argument from the caller.
> 
> Either way, at a minimum I'd want all `percent_s` to be consistent. You could go as far as extracting thse into a separate method in a mixin, ala https://github.com/dblock/open-weather-ruby-client/blob/master/lib/open_weather/models/mixins/temp.rb.

+1 Agree with the consitancy